v21.2/cluster-setup-troubleshooting.md

```diff
@@ -490,56 +513,18 @@ Common reasons for node liveness issues include:
 
 - Heavy I/O load on the node. Because each node needs to update a liveness record on disk, maxing out disk bandwidth can cause liveness heartbeats to be missed. See also: [Capacity planning issues](#capacity-planning-issues).
 - Outright I/O failure due to a [disk stall](#disk-stalls). This will cause node liveness issues for the same reasons as listed above.
-- Any [Networking issues](#networking-issues) with the node.
+- [Insufficient CPU for the workload](#cpu-is-insufficient-for-the-workload). This can eventually cause nodes to miss their liveness heartbeats and become unresponsive.
+- [Networking issues](#networking-issues) with the node.
 
 The [DB Console][db_console] provides several ways to check for node liveness issues in your cluster:
 
-- [Check node heartbeat latency](#check-node-heartbeat-latency)
-- [Check node liveness record last update](#check-node-liveness-record-last-update)
-- [Check command commit latency](#check-command-commit-latency)
+- [Check node heartbeat latency](common-issues-to-monitor.html#node-heartbeat-latency)
+- [Check command commit latency](common-issues-to-monitor.html#command-commit-latency)
 
 {{site.data.alerts.callout_info}}
-For more information about how node liveness works, see [the architecture documentation on the replication layer](architecture/replication-layer.html#epoch-based-leases-table-data).
+For more information about how node liveness works, see [Replication Layer](architecture/replication-layer.html#epoch-based-leases-table-data).
 {{site.data.alerts.end}}
 
-### Check node heartbeat latency
-
-To check node heartbeat latency:
-
-1. In the [DB Console][db_console], select the **Metrics** tab from the left-hand side of the page.
-
-2. From the metrics page, select **Dashboard: Distributed** from the dropdown at the top of the page.
-
-3. Scroll down the metrics page to find the **Node Heartbeat Latency: 99th percentile** and **Node Heartbeat Latency: 90th percentile** graphs.
-
-**Expected values for a healthy cluster**: Less than 100ms in addition to the network latency between nodes in the cluster.
-
-### Check node liveness record last update
```

Comment:
I removed this section (just the "node liveness record last update" part) because, following a convo with @dhartunian, this seems to be checking the last update to node **status** in the DB Console, not liveness record. I can make a correction in case we do have a way to check for the latter.